### PR TITLE
vpc peering info module

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

File: plugins/modules/vpc_peering_info.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  - Get a generator of vpc peerings info from the OTC.
+options:
+  name:
+    description:
+      - Peering connection name.
+    type: str
+  status:
+    description:
+      - Peering connection status.
+    choices: [PENDING_ACCEPTANCE, REJECTED, EXPIRED, DELETED, ACTIVE]
```
Comment (gtema) [inline plugins/modules/vpc_peering_info.py:31]: please use lowercase for the values

File: plugins/modules/vpc_peering_info.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        status=dict(required=False, choices=['PENDING_ACCEPTANCE', 'REJECTED', 'EXPIRED', 'DELETED', 'ACTIVE']),
+        tenant_id=dict(required=False),
+        vpc_id=dict(required=False),
+    )
+
+    def run(self):
+
+        name_filter = self.params['name']
+        status_filter = self.params['status']
+        tenant_id_filter = self.params['tenant_id']
```
Comment (gtema) [inline plugins/modules/vpc_peering_info.py:106]: we don't use tenant_id, but the project_id instead all over the code

File: plugins/modules/vpc_peering_info.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        vpc_id_filter = self.params['vpc_id']
```
Comment (gtema) [inline plugins/modules/vpc_peering_info.py:107]: and this one is router_id

**Review by gtema — CHANGES_REQUESTED** [review]

File: plugins/modules/vpc_peering_info.py (hunk 1 above)
Comment (Polina-Gubina) [reply plugins/modules/vpc_peering_info.py:31]: but response is in uppercase, so when I use lowercase  this filter condition doesn't work

Comment (Polina-Gubina) [reply plugins/modules/vpc_peering_info.py:31]: or do you mean something else?

Comment (gtema) [reply plugins/modules/vpc_peering_info.py]: you accept them in lowercase and convert to upper case when invoking API

**Review by gtema — APPROVED** [review]

**Pull request merged**